### DISCUSSION TITLE
Daily 28/9

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#39] [General]
Ayer practiqué Python e hice el update del documento del IEEE, solo queda terminar una sección además de que hay personas que no terminaron llenaron sus datos todavía ahí. También propuse un potencial [mapa del sitio](https://github.com/rpgrca/proyecto-ispc-fullstack/discussions/37) donde tenemos unas 9-10 páginas pesadas más las de login/registración/recordatorio de clave, así que con ese listado voy a armar historias de usuario para cada una de ellas.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve con Python en la plataforma de Cisco. Cree una US de Martillero. Además complete el cuestionario de SGC. No pude dedicarle tiempo a CSS.
Hoy estaré leyendo el nuevo capítulo de SGC y haciendo repaso de CSS.

**Reply by NataliaAlvarezIspc** [reply]
Agregue algunos items a la sección de restricciones del documentos. Revisen y agreguen si se les ocurre alguna más.

**Reply by rpgrca** [reply]
Quité la de concurrencia por si acaso, no lo vamos a hacer en esta primera parte y no quiero ni la menor chance de que nos digan que eso que habíamos puesto no lo hicimos! El resto está perfecto!

**Comment by rocappellano** [comment]
Realice el examen de inglés, en un rato me pondré a estudiar teoría de Programador FullStack

**Comment by Brian-Almada** [comment]
Ayer estudie Python. Hoy repaso de ingles y hacer el examen. Luego hacer actividad 2 de SGC.

**Comment by IngridAlcocer05** [comment]
Ayer realice la actividad 1 de CMS. Hoy practique html y realice la actividad de hosting y dominio.

**Comment by Thiagox749** [comment]
Hoy realice el examen de ingles, realice actividades de CMS y termine el modulo 3 de python.
